src/app/_components/Card.tsx

```diff
@@ -114,18 +107,18 @@ Card.Image = function ImageComponent({
 
   if (isStaticImage) {
     return (
-      <Image
+      <SmartImage
         {...commonProps}
         className={clsx(commonProps.className, 'aspect-video')}
-        src={image.data}
+        src={(image.data as StaticImageData).src}
```

Comment:
Typescript is complaining here although we have a `isStaticImage` condition (which check `data` in the `image`)

Reply:
It should just be `image.src` no? `image.data` doesn't exist on `SmartImageProps`

Reply:
Sorry not sure I understand 😕 `StaticImages` have data key, like this: 

```
{
  data: {
    src: '/_next/static/media/022624-ff-anualreport.b5deae8f.png',
    height: 1536,
    width: 3072,
    blurDataURL: '/_next/image?url=%2F_next%2Fstatic%2Fmedia%2F022624-ff-anualreport.b5deae8f.png&w=8&q=70',
    blurWidth: 8,
    blurHeight: 4
  },
  alt: '',
  sizes: '(min-width: 640px) 710px, (min-width: 768px) 980px, (min-width: 1024px) 480px, 100vw'
}
```

Ah so this should actually be `image.data`...